docs/administrative-requirements.md

```diff
@@ -23,66 +23,61 @@ Preparing to connect your system to Enterprise Event Bus may require additional
 * The client system has approved PTA and PIA documents.
 * Depending on the client system, an ESECC request and/or WASA scan may also be required.
 
-## VASI and eMASS 
-
-The client system should either have or be in the process of procuring: 
-
-* a VASI ID, which identifies your system in VA, and
-* an eMASS ID
-
-**Note**: The VASI and eMASS Processes have merged and are now accomplished via a single LEAF intake (formerly GRC intake) request: [Unified System Registry Intake (must be on VA network to view)](https://leaf.va.gov/NATIONAL/103/cybersecurity_request_portal/).
-
-### VASI
+## Authority to Operate (ATO)
 
-The VA Systems Inventory (VASI) is intended to be a registry of all applications in use at VA. 
+ATO, or “Authority to Operate,” indicates that your system has been evaluated by VA and given permission to deploy code in production.
 
-The Enterprise Event Bus VASI ID is 3325, and [this is our entry (must be on VA network to view)](https://vaww.vear.ea.oit.va.gov/system_and_application_domain_defs_system_381405.htm) in the VASI registry. You may find your system’s VASI ID by visiting the [VA System Inventory (must be on VA network to view)](https://vaww.vear.ea.oit.va.gov/).
+Teams that have an active ATO should review it to understand the implications of integrating with other systems.
 
 **For non-LHDI clients**:
 
-* If a prospective client system does not have a VASI ID, it will need to submit a LEAF intake request to get one. 
+ATO is evaluated once eMASS system registration has been completed and approved. Teams that do not have an active ATO should begin the process as early as possible in the Enterprise Event Bus integration process.
 
 **For new LHDI clients**:
```

Comment:
```suggestion
**For new LHDI clients**: For clients who are new tenants on LHDI
```

Reply:
Do we have different guidance for clients who are new to Event Bus *and* are existing LHDI tenants?

I made the suggestion to change "For new LHDI clients" "For clients who are new tenants on LHDI" to help draw that distinction. I think what we are going for here is that we have four scenarios that a new Event Bus Client can fall into:

1. Outside LHDI, already has an ATO.
2. Outside LHDI, does not have an ATO.
3. LHDI tenant, already has an ATO (which would be via cATO, like Event Bus).
4. LHDI tenant, does not have an ATO.

Scenarios 1 and 2 are covered in the **For non-LHDI clients**: text.

Scenario 4 is what I think we are addressing in the **For new LHDI clients**: text.

Scenario 3 is something that is missing.

Reply:
I think really the two scenarios that are relevant are, teams with an ATO and teams without. We really only care if they are without, so if that's the case there are two options, non-LHDI, and new to LHDI. We could add a header around these existing options that says something like, "For teams without an ATO".

Reply:
This has been updated for all sections, including some changes to formatting and clean-up.